components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-profiles/reporting-profiles.component.ts

```diff
@@ -80,24 +84,46 @@ export class ReportingProfilesComponent implements OnInit, OnDestroy {
   }
 
   addFilter(profile) {
-    const filter = {type: {name: 'profile'}, value: {...profile, text: profile.name}};
-    this.reportQuery.addFilter(filter);
+    const typeName = 'profile_id';
+    const title = `${profile.title}, v${profile.version}`;
+
+    this.reportQuery.setFilterTitle(typeName, profile.id, title);
```

Comment:
This allows the filter bar to display the name of the profile instead of the ID.